schemas/common/externalIdsHolder.json

```diff
@@ -7,6 +7,10 @@
     "instanceId": {
       "description": "instance id",
       "$ref": "uuid.json"
+    },
+    "instanceHrid": {
+      "description": "instance hrid",
+      "$ref": "uuid.json"
```

Comment:
Hrid is a human readable id, not a uuid (nobody can read this abracadabra), it's just a string, no validation needed

Reply:
@KaterynaSenchenko Thank you! Changed. could you please verify it?